README.md

```diff
@@ -32,37 +38,59 @@ json:
   like to generate.
 
 The example config below sets up the following simulation:
-* Connect to a single node `Alice` to generate activity.
+* Connect to `Alice` running LND to generate activity.
+* Connect to `Bob` running CLN to generate activity.
 * Dispatch 100 msat payments from `Alice` to `Carol` every 10 seconds.
-* Dispatch 100000 msat payments from `Alice` to `Eve` every 5 minutes.
+* Dispatch 100000 msat payments from `Bob` to `Alice` every 5 minutes.
```

Comment:
These two descriptions do not match the provided activity.

`Alice` is actually sending `Carol` 2000 msat every 1 sec
`Bob` is actually sending `Alice` 140000 msat every 50 sec